packages/react-instantsearch-hooks-dom/src/ui/Hits.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+import type { Hit } from 'instantsearch.js';
+import type { JSXElementConstructor } from 'react';
+
+export type HitsProps<THit> = React.ComponentProps<'div'> & {
+  hits: THit[];
+  hitComponent?: JSXElementConstructor<{ hit: THit }>;
+};
+
+function DefaultHitComponent({ hit }: { hit: Hit }) {
+  return (
+    <div
+      style={{
+        borderBottom: '1px solid #bbb',
+        paddingBottom: '5px',
+        marginBottom: '5px',
```

Comment:
Why are those needed?

Reply:
I guess the border is otherwise really close to the text? This is the same default component as in the current React InstantSearch, I didn't question it

Reply:
Definitely makes sense if there's a border to have padding, but I was questioning the border as well.

Ultimately we don't care much since this won't be used in production, but changing it is _technically_ a breaking change and we'll soon have new UI guidelines to apply. Maybe we can stay conservative on this one and just make sure it doesn't generate a horizontal scrollbar, as the `word-break` does, wdyt?